quickstart-latest.yml

```diff
@@ -6,3 +6,6 @@ services:
 
   kratos:
     image: oryd/kratos:latest-sqlite
+
+  kratos-selfservice-ui-node:
+    image: oryd/kratos-selfservice-ui-node:latest
```

Comment:
I don’t think we publish a latest tag for this image! Unless we do, this would be very outdated (like version 0.1)